metadata/location/utah_address_points

```diff
@@ -80,6 +80,6 @@ Address
 
 ### Update Schedule
 
-UGRC works with each county or municipality's MAL authority to continually improve the address point data. Counties have been placed on an update schedule depending on numerous factors, including rates of development and population growth. Densely populated counties, such as Salt Lake, Utah, and Weber, are updated monthly, while less populous counties may be updated quarterly or semi-annually.
+UGRC works with each county's MAL authority to continually improve the address point data. Counties have been placed on an update schedule depending on numerous factors, including rates of development and population growth. Densely populated counties, such as Salt Lake, Utah, and Weber, are updated monthly, while less populous counties may be updated quarterly or semi-annually.
```

Comment:
Lets mention all 5 counties that get monthly updates (Davis, Salt Lake, Utah, Washington, and Weber).